examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -378,7 +378,7 @@ def get_separation(rx_band, tx_band):
 show_separation_table(rx2_results, title='Separation for Bluetooth and WiFi (MHz)')
 
 # Need this to ensure plots don't close
-input("Press [enter] to continue.")
+#input("Press [enter] to continue.")
```

Comment:
@maxcapodi78 Is there a way to detect from within a script when the documentation build is being performed? There are several user-interaction features we'd like to enable in the example, but we have to comment them out to satisfy the doc build. If there were an environment variable or something we could check for we could make the example function as intended _and_ allow the doc build to complete successfullly.

Reply:
yes
check PYAEDT_DOC_GENERATION env variable. If it is setup and is set to 1, the documentation is building. If doesn't exists it means it's user run